### PR TITLE
1 - Explanation & Project Structure

### PR DESCRIPTION
**Pull request by MarcoMartins2417**
- Created Number Trivia App

### CODE REVIEW

File: number_trivia_app/android/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,71 @@
+def localProperties = new Properties()
```
Comment (andre-exd) [inline number_trivia_app/android/app/build.gradle:1]: Please consider removing platforms that are not being used. Android and iOS might be the ones that could be kept

File: number_trivia_app/lib/main.dart [hunk 2]
```diff
@@ -0,0 +1,115 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  // This widget is the root of your application.
```
Comment (andre-exd) [inline number_trivia_app/lib/main.dart]: Remove comments, please

File: number_trivia_app/pubspec.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
+name: number_trivia_app
+description: A new Flutter project.
+
+# The following line prevents the package from being accidentally published to
```
Comment (andre-exd) [inline number_trivia_app/pubspec.yaml]: Consider removing all unnecessary comments

File: number_trivia_app/test/widget_test.dart [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+// utility in the flutter_test package. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:number_trivia_app/main.dart';
+
+void main() {
```
Comment (andre-exd) [inline number_trivia_app/test/widget_test.dart]: Consider removing this file

**Review by andre-exd — CHANGES_REQUESTED** [review]

File: number_trivia_app/pubspec.yaml (hunk 3 above)
Comment (MarcoMartins2417) [reply number_trivia_app/pubspec.yaml]: This change is done in the following pull requests!

File: number_trivia_app/test/widget_test.dart (hunk 4 above)
Comment (MarcoMartins2417) [reply number_trivia_app/test/widget_test.dart]: This file is removed on the following pull requests!

File: number_trivia_app/pubspec.yaml (hunk 3 above)
Comment (andre-exd) [reply number_trivia_app/pubspec.yaml]: Please do it in this branch anyway!

File: number_trivia_app/test/widget_test.dart (hunk 4 above)
Comment (andre-exd) [reply number_trivia_app/test/widget_test.dart]: Remove it anyway. The purpose is to force conflicts on future PRs (Pull Requests)

File: number_trivia_app/linux/flutter/generated_plugin_registrant.cc [hunk 5]
```diff
@@ -0,0 +1,11 @@
+//
```
Comment (andre-exd) [inline number_trivia_app/linux/flutter/generated_plugin_registrant.cc]: Seems you re-added unnecessary files. Please remove files from linux, macos and windows

**Review by andre-exd — CHANGES_REQUESTED** [review]

**Review by andre-exd — APPROVED** [review]

**Pull request merged**